src/controllers/commentController.js

```diff
@@ -20,11 +22,33 @@ export default class CommentController {
       const { user } = req;
       const { comment } = req.body;
       const { requestId } = req.params;
+      const { email } = req.managerDetails;
       const newComment = await db.Comments.create({
         requestId,
         commmentOwner: user.id,
         comment,
       });
+      if (user.role === 'manager') {
+        const notifyUser = await notifService.createNotif(req.request.userId, req.request.email, 'your manager posted a comment', '#');
+        const content = {
+          intro: req.__(notifyUser.content),
+          instruction: req.__(`To view this ${newComment.comment} click below`),
```

Comment:
could add internalization for these strings

Reply:
it is done.